lte/gateway/c/core/oai/lib/s6a_proxy/s6a_client_api.hpp

```diff
@@ -24,7 +24,7 @@ extern "C" {
 #endif
 
 #include "lte/gateway/c/core/oai/lib/itti/intertask_interface.h"
-#include "lte/gateway/c/core/oai/include/s6a_messages_types.h"
+#include "lte/gateway/c/core/oai/include/s6a_messages_types.hpp"
 
 /**
  * s6a_purge is an asynchronous call that forwards S6a PU to Federation Gateway
```

Comment:
All these fun(), s6a_purge_ue, s6a_authentication_info_req and s6a_update_location_req need not be under extern C